app/src/main/java/io/pumpkinz/pumpkinreader/model/Comment.java

```diff
@@ -13,13 +15,17 @@
     Comment parentComment;
     List<Comment> childComments;
 
-    public Comment() {}
+    public Comment() {
+        this.kids = new ArrayList<>();
+        this.childComments = new ArrayList<>();
```

Comment:
oh well, the naming from HN again.
`kids` is commentIds.
`childComments` is the actual `Comment` object.